shardingsphere-infra/shardingsphere-infra-common/src/main/java/org/apache/shardingsphere/infra/metadata/schema/builder/TableMetaDataBuilder.java

```diff
@@ -55,73 +55,52 @@
      * @throws SQLException SQL exception
      */
     public static Optional<TableMetaData> build(final String tableName, final SchemaBuilderMaterials materials) throws SQLException {
-        Optional<TableMetaData> tableMetaData = load(tableName, materials);
-        return tableMetaData.map(optional -> decorate(tableName, optional, materials.getRules()));
+        return Optional.ofNullable(load(Collections.singleton(tableName), materials).getOrDefault(tableName, null)).map(tableMetaData -> decorate(tableName, tableMetaData, materials.getRules()));
```

Comment:
@tuichenchuxin Please extract a variable to improve code readability. BTW, `getOrDefault` is better replaced by `get`.